quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testExportHelp.unix.approved.txt

```diff
@@ -41,8 +59,31 @@ Database:
 --db-username <username>
                      The username of the database user.
 
+Transaction:
+
+--transaction-xa-enabled <true|false>
+                     If set to false, Keycloak uses a non-XA datasource in case the database does
+                       not support XA transactions. Default: true.
+
+Feature:
+
+--features <feature> Enables a set of one or more features. Possible values are: account-api,
+                       account2, admin-api, admin-fine-grained-authz, admin2, authorization, ciba,
+                       client-policies, client-secret-rotation, declarative-user-profile, docker,
+                       dynamic-scopes, fips, impersonation, js-adapter, kerberos, map-storage,
+                       openshift-integration, par, preview, recovery-codes, scripts,
+                       step-up-authentication, token-exchange, update-email, web-authn.
+--features-disabled <feature>
+                     Disables a set of one or more features. Possible values are: account-api,
+                       account2, admin-api, admin-fine-grained-authz, admin2, authorization, ciba,
+                       client-policies, client-secret-rotation, declarative-user-profile, docker,
+                       dynamic-scopes, fips, impersonation, js-adapter, kerberos, map-storage,
+                       openshift-integration, par, preview, recovery-codes, scripts,
+                       step-up-authentication, token-exchange, update-email, web-authn.
+
 Vault:
 
+--vault <provider>   Enables a vault provider. Possible values are: file.
```

Comment:
Do make sense vault options on import/export?

Reply:
We've never tested export/import with different settings from a normal Keycloak instance, so I wouldn't know if we depend on the vault for federation credentials, and if those would be referenced during an export or import. So I'd rather keep them as options.

Reply:
During import/export we are not resolving values from vaults but using their raw value. I don't think we need it.